config/jobs/ppc64le-cloud/bazel/bazel-postsubmit.yaml

```diff
@@ -0,0 +1,38 @@
+postsubmits:
+  ppc64le-cloud/test-infra:
+    - name: postsubmit-bazel-job
+      decorate: true
+      branches:
+        - master
+      extra_refs:
+        - base_ref: v0.1.2
+          org: vmware-tanzu
+          repo: buildkit-cli-for-kubectl
+      spec:
+        serviceAccountName: build-img
```

Comment:
We might not need to run this job with `build-img` service account from IKS as we are not framing the KUBECONFIG in job.
@mkumatag please confirm.

Reply:
Make sense..